test/fail_compilation/udaparams.d

```diff
@@ -0,0 +1,24 @@
+/*
+TEST_OUTPUT:
+---
+fail_compilation/udaparams.d(15): Error: variadic parameter cannot has UDAs
+fail_compilation/udaparams.d(16): Error: variadic parameter cannot has UDAs
```

Comment:
This is inconsistent with the error message on the very next line, which uses `user defined attributes` as opposed to `UDAs`. You should use either one or the other, but not both.